src/io/ioexternalbus.h

```diff
@@ -39,11 +42,24 @@ class IOExternalBus : public IOBase {
 protected:
     virtual void parameterChanged(unsigned) override;
 
+private slots:
+    void connectButtonTriggered();
+    // void sktConnected();
+    // void sktRead();
+
 private:
+    uint32_t ByteSize;
+    int32_t send_cmd(const uint32_t cmd, const uint32_t payload_size = 0, const char* payload = nullptr);
+    int32_t recv_cmd(cmd_header_t* cmd_header);
```

Comment:
Any reason why this is a pointer and not a reference?

Reply:
changed to reference.